### PR TITLE
(PC-19754) fix(ConfirmDeleteProfile): Update wording

### PR DESCRIPTION
**Pull request by aliraiki**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-19754

## Checklist

I have:

- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [ ] Written **unit tests** native (and web when implementation is different) for my feature.
- [x] Added a **screenshot** for UI tickets or deleted the screenshot section if no UI change
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion](1)

## Screenshots

**delete** *if no UI change*

| Platform         | Before | After |
| :--------------- | :----: | :---: |
| iOS              |        |     Simulator Screen Shot - iPhone 13 - 2023-01-23 at 16 04 01  |
| Android          |        |   Screenshot_20230123_160433    |
| Desktop - Chrome |        |  Capture d’écran 2023-01-23 à 16 03 56     |
| Desktop - Safari |        |       |

[1]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

**Comment by aliraiki** [comment]
Les stories cassées / celles qui ont évoluées sont déjà sur master...

File: src/features/profile/pages/DeleteProfile/ConfirmDeleteProfile.tsx [hunk 1]
```diff
@@ -54,7 +56,15 @@ export function ConfirmDeleteProfile() {
       <Content>
         <Typo.ButtonText>Les conséquences&nbsp;:</Typo.ButtonText>
         <VerticalUl>
-          <BulletListItem text="tes réservations seront annulées et supprimées" />
+          <BulletListItem text="tes réservations sont annulées et suppriméestes réservations sont annulées sauf pour certains cas précisées dans les ">
```
Comment (matthias-prevost) [inline src/features/profile/pages/DeleteProfile/ConfirmDeleteProfile.tsx:59]: Petit problème de wording ici, tu as du oublié de supprimer le début de la phrase :)

File: __snapshots__/features/profile/pages/DeleteProfile/ConfirmDeleteProfile.native.test.tsx.native-snap [hunk 2]
```diff
@@ -204,7 +204,82 @@ Array [
                 ]
               }
             >
-              tes réservations seront annulées et supprimées
+              tes réservations sont annulées et suppriméestes réservations sont annulées sauf pour certains cas précisées dans les
```
Comment (matthias-prevost) [inline __snapshots__/features/profile/pages/DeleteProfile/ConfirmDeleteProfile.native.test.tsx.native-snap:207]: Du coup le snapshot est à update

File: src/features/profile/pages/DeleteProfile/ConfirmDeleteProfile.tsx (hunk 1 above)
Comment (aliraiki) [reply src/features/profile/pages/DeleteProfile/ConfirmDeleteProfile.tsx:59]: Oups

**Review by matthias-prevost — APPROVED** [review]

**Pull request merged**